tests/kohana/ExceptionTest.php

```diff
@@ -78,17 +78,18 @@ public function test_constructor($arguments, $expected_message, $expected_code)
 	 */
 	public function provider_text()
 	{
-		return array(
-			array(new Kohana_Exception('foobar'), $this->dirSeparator('Kohana_Exception [ 0 ]: foobar ~ SYSPATH/tests/kohana/ExceptionTest.php [ '.__LINE__.' ]')),
-		);
+		return [
+			[new Kohana_Exception('foobar'), $this->dirSeparator('Kohana_Exception [ 0 ]: foobar ~ SYSPATH/tests/kohana/ExceptionTest.php [ '.__LINE__.' ]'),],
+		];
```

Comment:
How about more like this just to make it safe for any future autoformat:
```suggestion
		$exception = new Kohana_Exception('foobar');
		$exception_line = __LINE__ - 1;
		return [
			[
				$exception,
				$this->dirSeparator(
					"Kohana_Exception [ 0 ]: foobar ~ SYSPATH/tests/kohana/ExceptionTest.php [$exception_line]"
				),
			],
		];
```